src/main/steps/children/social-worker/content.test.ts

```diff
@@ -0,0 +1,178 @@
+  expect(generatedContent.errors).toEqual(errors);
+};
+
+const commonContent = { language: EN } as CommonContent;
+
+describe('social worker content', () => {
+  it('should return the correct content for language = en', () => {
+    langAssertions(EN, enContent);
+  });
+
+  it('should return the correct content for language = cy', () => {
+    langAssertions(CY, cyContent);
+  });
+
+  it('should have an socialWorkerName text input field', () => {
+    const generatedContent = generateContent(commonContent);
+    const form = generatedContent.form as FormContent;
+    const fields = form.fields as FormFields;
+    const socialWorkerName = fields.socialWorkerName;
+
+    expect(socialWorkerName.type).toBe('text');
+    expect((socialWorkerName.label as Function)(generateContent(commonContent))).toBe(enContent.socialWorkerName);
+
+    (socialWorkerName.validator as Function)('MockName');
+    expect(isFieldFilledIn).toHaveBeenCalledWith('MockName');
+  });
+
+  it('should have an socialWorkerPhoneNumber text input field', () => {
+    const generatedContent = generateContent(commonContent);
+    const form = generatedContent.form as FormContent;
+    const fields = form.fields as FormFields;
+    const socialWorkerPhoneNumber = fields.socialWorkerPhoneNumber;
+
+    expect(socialWorkerPhoneNumber.type).toBe('text');
+    expect((socialWorkerPhoneNumber.label as Function)(generateContent(commonContent))).toBe(
+      enContent.socialWorkerPhoneNumber
+    );
+
+    (socialWorkerPhoneNumber.validator as Function)('MockPhoneNumber');
+    expect(isFieldFilledIn).toHaveBeenCalledWith('MockPhoneNumber');
+  });
+
+  it('should have an socialWorkerEmail text input field', () => {
+    const generatedContent = generateContent(commonContent);
+    const form = generatedContent.form as FormContent;
+    const fields = form.fields as FormFields;
+    const socialWorkerEmail = fields.socialWorkerEmail;
+
+    expect(socialWorkerEmail.type).toBe('text');
+    expect((socialWorkerEmail.label as Function)(generateContent(commonContent))).toBe(enContent.socialWorkerEmail);
+
+    (socialWorkerEmail.validator as Function)('MockEmail');
+    expect(isFieldFilledIn).toHaveBeenCalledWith('MockEmail');
+  });
+
+  it('should have an socialWorkerTeamEmail text input field', () => {
+    const generatedContent = generateContent(commonContent);
+    const form = generatedContent.form as FormContent;
+    const fields = form.fields as FormFields;
+    const socialWorkerTeamEmail = fields.socialWorkerTeamEmail;
+
+    expect(socialWorkerTeamEmail.type).toBe('text');
+    expect((socialWorkerTeamEmail.label as Function)(generateContent(commonContent))).toBe(
+      enContent.socialWorkerTeamEmail
+    );
+
+    const errors = generatedContent.errors as any;
+    expect(errors.socialWorkerTeamEmail.invalid).toEqual(
+      'Enter an email address in the correct format, like name@example.com'
```

Comment:
content testing should not be here

Reply:
Please run test:coverage and see the report

Reply:
--------------------------------------------------------|---------|----------|---------|---------|-----------------------------------------------------
File                                                    | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s                                   
--------------------------------------------------------|---------|----------|---------|---------|-----------------------------------------------------
All files                                               |   96.02 |    85.21 |    92.7 |   95.73 |
 main/steps/children/social-worker                      |     100 |    83.33 |     100 |     100 |                                                     
  content.ts                                            |     100 |    83.33 |     100 |     100 | 88